app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,81 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.exceptions.ProductNotFoundException;
+import javax.transaction.Transactional;
+import org.springframework.stereotype.Service;
+
+/**
+ * 상품 목록 관리를 담당합니다.
+ */
+@Service
+@Transactional
+public class ProductService {
+
+    ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 상품을 추가합니다.
+     *
+     * @param product 추가할 상품
+     * @return 추가된 상품
+     */
+    public Product createProduct(Product product) {
+        return productRepository.save(product);
```

Comment:
javadoc에서는 리턴값만 설명해줘도 평타를 칩니다.

```suggestion
    /**
     * 상품을 추가하고, 리턴합니다.
     *
     * @param product 추가할 상품
     * @return 추가된 상품
     */
    public Product createProduct(Product product) {
        return productRepository.save(product);
```

Reply:
리턴한다는 설명을 빼먹었네요..